Leetcode/EigenDemo/eigen_practice.hpp

```diff
@@ -0,0 +1,25 @@
+//
+//  eigenPractice.hpp
+//  EigenDemo
+//
+//  Created by dsh on 2021/7/18.
+//
+
+#ifndef EIGENDEMO_EIGEN_PRACTICE_H_
+#define EIGENDEMO_EIGEN_PRACTICE_H_
+
+#include <iostream>
+#include <stdio.h>
+
+#include "Eigen/Eigen"
+
+constexpr int MATRIX_SIZE = 50;
```

Comment:
```suggestion
constexpr int kMatrixSize = 50;
```

https://google.github.io/styleguide/cppguide.html#Constant_Names